Jenkinsfile

```diff
@@ -183,4 +184,24 @@ def evaluateJson(String json, String gpath){
     def ojson = new groovy.json.JsonSlurper().parseText(json)
     //evaluate gpath as a gstring template where $json is a parsed json parameter
     return new groovy.text.GStringTemplateEngine().createTemplate(gpath).make(json:ojson).toString()
-}
\ No newline at end of file
+}
+
+def notifySlack(String buildStatus) {
+
+    // build status of null means successful
+    buildStatus =  buildStatus ?: 'SUCCESS'
+
+    // Default values
+    def colorCode = '#FF0000'
+    def summary = "${buildStatus}: $JOB_NAME: \n<$BUILD_URL|Build #$BUILD_NUMBER> - $currentBuild.displayName"
+
+    // Override default values based on build status
+    if (buildStatus == 'SUCCESS') {
+        colorCode = '#00FF00'
+    } else {
+        summary = "@channel " + summary
+    }
+
+    // Send notifications to channel
+    slackSend (color: colorCode, message: summary)
```

Comment:
where is `slackSend` defined?

Reply:
its a plugin I added to jenkins.

Reply:
i see, the plugin must define the rook slack url and the channel name